### ISSUE TITLE
Page crash when load Adbanner

### ISSUE THREAD

**Issue by banhthanhvi** [marcojak/MauiMTAdmob#19]
xmlns:admod="clr-namespace:Plugin.MauiMTAdmob.Controls;assembly=Plugin.MauiMtAdmob"
<admod:MTAdView Grid.Row="2" x:Name="myAds" AdSize="AnchoredAdaptive" IsVisible="true" 
                            AdsId="{Binding AdbannerId}" VerticalOptions="End" 
                            HeightRequest="{OnIdiom Phone=50, Tablet=90, Desktop=90}" />

My MAUI page has crash when add above code.
I use:
   - MauiMTAdmob 1.0.2 
   - .NET 7.0
   - VS 2022 17.4.3
   - Xcode 14.1
It ran ok with:
   - MauiMTAdmob 1.0.1 
   - .NET 6.0

Do I miss something? 
Thank you.

**Comment by banhthanhvi** [comment]
It crashes on build iOS release.

**Comment by DeerSteak** [comment]
I think this is a Xamarin.Google.iOS.MobileAds issue - if you call MobileAds.SharedInstance.Start in AppDelegate, it crashes because MobileAds.SharedInstance is null. Since this plugin relies on the Xamarin library, my guess is it's quietly doing the same thing. 

You can test it by making this your AppDelegate:

```
[Register("AppDelegate")]
public class AppDelegate : MauiUIApplicationDelegate
{
    protected override MauiApp CreateMauiApp() => MauiProgram.CreateMauiApp();

    public override bool FinishedLaunching(UIApplication application, NSDictionary launchOptions)
    {
        //MobileAds.SharedInstance.Start(AdsCompletionHandler);

        return base.FinishedLaunching(application, launchOptions);
    }

    private void AdsCompletionHandler(InitializationStatus status) { }
}

```

**Comment by Martinedo** [comment]
was there any progress on this issue? it still occurs

**Comment by marcojak** [comment]
Does the sample work for you?

**Comment by Martinedo** [comment]
I tried the sample and it worked. Then I compared it with my solution and the problem was not with the library or the code, but with my target version of iOS, which was set too low on 11, I tested it on the same as in the sample 14.2 and it works! thank you :)

```
<SupportedOSPlatformVersion Condition="$([MSBuild]::GetTargetPlatformIdentifier('$(TargetFramework)')) == 'ios'">14.2</SupportedOSPlatformVersion>
```

I tested different versions and iOS 12 is not supported, so iPhone 5S and 6 are unlucky. every newer iphone should be ok.
